src/Uno.UI.Runtime.Skia.Wpf/Input/WpfKeyboardInputSource.cs

```diff
@@ -156,8 +136,11 @@ private static VirtualKeyModifiers GetKeyModifiers(ModifierKeys modifierKeys)
 		return modifiers;
 	}
 
-	private VirtualKey ConvertKey(Key key)
+	private static VirtualKey ConvertKey(Key key)
 	{
 		return (VirtualKey)System.Windows.Input.KeyInterop.VirtualKeyFromKey(key);
 	}
+
+	[DllImport("User32.dll")]
```

Comment:
Could this be LibraryImport?

Reply:
@Youssef1313  I find your PR (https://github.com/unoplatform/uno/pull/16556) will replace all DllImport with LibraryImport. But I find the `InputHelper.MapVirtualKey` still use DllImport. Should I use LibraryImport here?

Reply:
My PR was focused on X11, but can do a follow-up everywhere in the codebase. I'm okay with it being DllImport. We can change later if needed.

Reply:
Thank you @Youssef1313